map/manage-operational-layers/src/main/java/com/esri/samples/manage_operational_layers/ManageOperationalLayersSample.java

```diff
@@ -184,6 +184,9 @@ public void start(Stage stage) {
       controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0,0,0,0.3)"), CornerRadii.EMPTY,
         Insets.EMPTY)));
       controlsVBox.getStyleClass().add("panel-region");
+      controlsVBox.setPadding(new Insets(10.0));
+      controlsVBox.setMaxWidth(200);
+      controlsVBox.setMaxHeight(260);
```

Comment:
Should we not remove these relevant lines from the `styles.css` file?
https://github.com/Esri/arcgis-runtime-samples-java/blob/42751302b377153ee1d5631b5374410eaf8d7822/map/manage-operational-layers/src/main/resources/manage_operational_layers/style.css#L1-L4

I'm also noticing the definition in the css is purely `.panel-region {` 
Have you tried `.panel-region .vbox {` by any chance? I don't have much faith in that suddenly working, buy knowing CSS it can sometimes fix things just by being more specific about which element to target with the rules.

Reply:
@JonLavi tried it but sadly not 👎  I don't think the sample viewer even reads the CSS files from the samples, so the label styling that does apply is from the sample viewer CSS files themselves is my theory. We've been given a bit of scope this sprint to look at the SV. @sclaridge FYI on this conversation.

For now, I'll remove the CSS anyway, as no point duplicating. Thanks!